### PR TITLE
{vis}[foss/2021a] freud-analysis v2.6.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)
Depends on:
- ~~#13353~~

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso EB_ARGS="--from-pr 13353"

File: easybuild/easyconfigs/f/freud-analysis/freud-analysis-2.6.2-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.20.1'),
+    ('scikit-build', '0.11.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('tbb', '2021.2.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/freud-analysis/freud-analysis-2.6.2-foss-2021a.eb:28]:
```suggestion
    ('tbb', '2020.3'),

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4012.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8
See https://gist.github.com/boegel/f679fcbf51f96fa71c821953931e84c5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**